### ISSUE TITLE
Database API: Example how to use "get_discussions_by_author_before_date" method

### ISSUE THREAD

**Issue by adcpm** [steemit/steem#204]
I'm looking to make working the method "get_discussions_by_author_before_date". I've seen these params : author, start_permlink, before_date, limit. I'm not sure what is the format, is there an example?

**Comment by aaroncox** [comment]
I'd love to know as well!

**Comment by ByronAP** [comment]
{"jsonrpc": "2.0","id":1,"method":"get_discussions_by_author_before_date", "params": ["accountname", "", "isoformateddatetime", 10]}

**Comment by ByronAP** [comment]
it doesn't seem to matter what the date is though, so I assume the call is not working 100%

**Comment by lukestokes** [comment]
I'm not sure if this is related to a problem I'm having, but when I go to my blog, I can't scroll past two weeks. That's really a bummer as I'd like for people to be able to see my older content, especially my introduceyourself post so they can get to know me better.

Here's the call I see in my web console:

```
SteemApi exec 0 get_discussions_by_author_before_date ( ["lukestokes", "a-temporary-banlist-solution-for-steemit-abuse", "1970-01-01T00:00:00", 20] )
```
But it returns nothing. Is that because it's looking for content before 1970? Seems there might be a bug in there somewhere.

**Comment by lukestokes** [comment]
Figured this might be a problem with the Steemit.com code sending in the wrong parameters to this method. When I search that codebase I found this:

https://github.com/steemit/steemit.com/search?utf8=%E2%9C%93&q=get_discussions_by_author_before_date

But when I click to view that bit of code, I get a 500 error from Github. Odd. If I could look at the code, maybe I could bring a solution instead of just a problem.

I've tried clearing my cookies and logging back in, but it still doesn't work.
